gravityflow/gravityflowchecklists
# Fix an issue with checklists icons on frontend

## Description
This PR fixes the issue of checklists icons missing on the frontend page. The [Gravity Forms PR#1589](https://github.com/gravityforms/gravityforms/pull/1589) worked to change the namespace of the legacy Font Awesome kit. We need to load the `gform_font_awesome-css` to ensure the Font Awesome icons continue to load for Checklists on the front-end.

This resolves [Issue#109](https://github.com/gravityflow/backlog/issues/109).

## Testing instructions
1. Create a Front-End page for Checklists view with the shortcode `[gravityflow page="checklists"].`
2. With the updated version of Gravity Forms (2.5), the checklists icons do not appear. They were appearing fine with the previous Gravity Forms version.
3. Switch to this branch and now check the frontend page to confirm the checklists icons appear fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
